docs/partial_source/design/building_blocks.rst

```diff
@@ -344,27 +344,29 @@ The compiler takes in any Ivy function, backend function, or composition, and re
    :align: center
    :width: 75%
 
-As an example, the following 3 pieces of code all compile to the exact same computation graph as shown:
+Let's look at a few examples, and observe the compiled graph of the requisite Ivy code, based on the desired backend.
```

Comment:
compiled graph "of the requisite Ivy code" doesn't really make sense. The middle example is not Ivy code at all for example. The point we are showing here is that the graph compiler can be applied either to "Ivy code" or to "native code".